numpy/__init__.pyi

```diff
@@ -2854,6 +2854,17 @@ class ndarray(_ArrayOrScalarCommon, Generic[_ShapeType, _DType_co]):
     @overload
     def __ior__(self: NDArray[_ScalarType], other: _RecursiveSequence) -> NDArray[_ScalarType]: ...
 
+    def bit_count(
+        self,
+        out: None | NDArray[Any] = ...,
+        *,
+        where: _ArrayLikeBool_co = ...,
+        casting: _CastingKind = ...,
+        order: _OrderKACF = ...,
+        dtype: DTypeLike = ...,
+        subok: bool = ...,
+    ) -> Any: ...
```

Comment:
```suggestion
    ) -> NDArray[Any]: ...
```
It seems the output types of `ndarray.bit_count` and `generic.bit_count` got swapped around.